### DISCUSSION TITLE
Import temperature from Exodus file

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#27666] [Q&A General]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I am currently trying to run a phase-field simulation to check the solidification of some grains. I would like to know if I can run a simulation only for the temperature, which creates an exodus file, and then use those temperatures for my solidification simulation. I was thinking that maybe there is a way to import the temperatures for each timestep from the thermal simulation to save time while doing the phase-field simulation. The mesh and timestep are equal for both simulations.

**Comment by GiudGiud** [comment]
Hello

You ll want to use a SolutionUserObject to load the file at the end of every time step

Then a SolutionAux to populate a temperature auxiliary variable

Guillaume
